targets/XPlatCppSdk/source/code/include/playfab/PlayFabAndroidHttpPlugin.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <playfab/PlayFabCallRequestContainer.h>
+#include <playfab/PlayFabPluginManager.h>
+#include <thread>
+#include <mutex>
+#include <atomic>
+
+namespace PlayFab
+{
+    /// <summary>
+    /// PlayFabAndroidHttpPlugin is an https implementation to interact with PlayFab services using WinHTTP.
+    /// </summary>
+    class PlayFabAndroidHttpPlugin : public IPlayFabHttpPlugin
+    {
+    public:
+        PlayFabAndroidHttpPlugin();
+        PlayFabAndroidHttpPlugin(const PlayFabAndroidHttpPlugin& other) = delete;
```

Comment:
>PlayFabAndroidHttpPlugin(const PlayFabAndroidHttpPlugin& other) [](start = 8, length = 63)

Do you need the copy assignment operator ?

Reply:
No, "PlayFabAndroidHttpPlugin& operator=(PlayFabAndroidHttpPlugin&& other) = delete;" is removing copy assignment operator.

---
In reply to: [273688270](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r273688270) [](ancestors = 273688270)